operator/pkg/ingress/annotations/annotations.go

```diff
@@ -33,6 +34,7 @@ const (
 	TCPKeepAliveProbeIntervalAnnotationAlias    = annotation.Prefix + "/tcp-keep-alive-probe-interval"
 	TCPKeepAliveProbeMaxFailuresAnnotationAlias = annotation.Prefix + "/tcp-keep-alive-probe-max-failures"
 	WebsocketEnabledAnnotationAlias             = annotation.Prefix + "/websocket"
+	TLSMinimumProtocolVersionAlias              = annotation.Prefix + "/tls-minimum-protocol-version"
```

Comment:
actually the recent push contains some extra changes, is it intentional ? 

I don't think the annotation here is working in term of setting minimum TLS version.

Reply:
Not intentional, sorry.. I was working on something else and thought I had stashed it. Updated commit.